docs/draft/keycard.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Keyacrd? (have no idea how to call it)
+version: 0.0.0
+status: Draft
+authors: Roman Volosovskyi <roman@status.im>
+redirect_from:
+  - /keycard.html
+---
+
+## Table of Contents
+
+1. [Abstract](#abstract)
+2. [Definitions](#definitions)
+3. [Multiaccount creation](#Multiaccount-creationrestoring)
+4. [Multiaccount restoring via pairing](#Multiaccount-restoring-via-pairing)
+5. [Multiaccount unlocking](#Multiaccount-unlocking)
+6. [Where are the keys stored?](#Where-are-the-keys-stored)
+
+## Abstract
+
+In this specification, we describe how Satus communicates with Keycard to create, store and use multiaccount.
+
+## Definitions
+
+
+| Term | Description |
+| -------- | ------------------ |
+|      |      |
+
+
+## Multiaccount creation/restoring
+### Creation and restoring via mnemonic
+1. `status-im.hardwallet.card/get-application-info` 
+    request: `nil`
+    response: `{"initialized?" false}`
+2. `status-im.hardwallet.card/init-card` 
+   request: `{:pin 123123}`
+   response: 
+   ```clojure
+   {"password" "nEJXqf6VWbqeC5oN", 
+    "puk" "411810112887", 
+    "pin" "123123"}
+   ```
+3. `status-im.hardwallet.card/get-application-info` 
+    request: `nil`
+    response: 
+    ```clojure
+    {"free-pairing-slots" 5, 
+     "app-version" "2.2", 
+     "secure-channel-pub-key" "04e70d7af7d91b8cd23adbefdfc242c096adee6c1b5ad27a4013a8f926864c1a4f816b338238dc4a04226ab42f23672585c6dca03627885530643f1656ee69b025", 
+     "key-uid" "", 
+     "instance-uid" "9f149d438988a7af5e1a186f650c9328", 
+     "paired?" false, 
+     "has-master-key?" false, 
+     "initialized?" true}
+    ```
+4. `status-im.hardwallet.card/pair` 
+   params: `{:password "nEJXqf6VWbqeC5oN"}`
+   response: `AAVefVX0kPGsxnvQV5OXRbRTLGI3k8/S27rpsq/lZrVR` (`pairing`)
+   
+5. `status-im.hardwallet.card/generate-and-load-keys` 
+    ```clojure
+    {:mnemonic "lift mansion moment version card type uncle sunny lock gather nerve math", 
```

Comment:
```suggestion
    {:mnemonic "lift mansion moment version card type uncle cloud lock gather nerve math", 
```

Reply:
but_why?.jpg

Reply:
it was a joke, pls dont take this suggestion seriously

Reply:
good :)